doc/INSTALL.md

```diff
@@ -22,74 +22,115 @@ For the following Linux distributions and package managers 0.8 packages are avai
-   ```
-
-### Debian-Buster
+```sh
+# Example for Ubuntu Focal
+apt-get install software-properties-common apt-transport-https
+echo "deb https://debs.libelektra.org/focal focal main" | sudo tee /etc/apt/sources.list.d/elektra.list
+```
 
-To use the Debian Buster repository of the latest builds from master put following lines in
-`/etc/apt/sources.list`:
+Or alternatively, you can use (if you do not mind many dependences just to add one line to a config file):
 
+```sh
+# Example for Ubuntu Focal
+sudo apt-get install software-properties-common apt-transport-https
+sudo add-apt-repository "deb https://debs.libelektra.org/focal focal main"
 ```
-deb     [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main
-deb-src [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main
+
+If you would like to use our bleeding-edge builds (master builds), append `-unstable` to `<DISTRIBUTION>`.
+
+The `etc/apt/source.list` entry must look like following: `deb https://debs.libelektra.org/<DISTRIBUTION> <DISTRIBUTION>-unstable main`
+
+E.g. `deb https://debs.libelektra.org/focal focal-unstable main`
+
+> NOTE: for Ubuntu Bionic the yamlcpp plugin is excluded due to missing dependencies and therefore the package `libelektra4-yamlcpp` is not available.
+
+### Fedora
+
+We also provide stable and bleeding-edge packages for Fedora 33.
+
+For the stable repository:
+
+```sh
+wget https://rpms.libelektra.org/fedora-33/libelektra.repo -O libelektra.repo;
+sudo mv libelektra.repo /etc/yum.repos.d/;
+sudo yum update
 ```
 
-Which can also be done using:
+Or alternatively you can use dnf to add this repo:
 
 ```sh
-sudo apt-get install software-properties-common apt-transport-https
-echo "deb     [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main" | sudo tee /etc/apt/sources.list.d/elektra.list
+dnf config-manager --add-repo https://rpms.libelektra.org/fedora-33/libelektra.repo
 ```
 
-Or alternatively, you can use (if you do not mind many dependences just to add one line to a config file):
+For our bleeding-edge builds append `-unstable` to the distribution name:
 
 ```sh
-sudo apt-get install software-properties-common apt-transport-https
-sudo add-apt-repository "deb     [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main"
+wget https://rpms.libelektra.org/fedora-33-unstable/libelektra.repo -O libelektra.repo;
+sudo mv libelektra.repo /etc/yum.repos.d/;
+sudo yum update
 ```
 
-If you want to rebuild Elektra from Debian unstable or
-our repositories, add a `deb-src` entry to `/etc/apt/sources.list`
-and then run:
+Or alternatively you can use dnf to add this repo:
 
 ```sh
-apt-get source -b elektra
+dnf config-manager --add-repo https://rpms.libelektra.org/fedora-33-unstable/libelektra.repo
 ```
 
 ### Install
 
 To get all packaged plugins, bindings and tools install:
 
 ```sh
+# For Debian based distributions
 apt-get install libelektra4-all
```

Comment:
We should pump this major version number (and SO_VERSION) to 5 to be ready for 1.0 release. This is also a good indicator for external people to indicate that this will be a "special" release with huge changes as @kodebach suggested.

Reply:
Do you mean with major version number the "4" in libelektra4-all, etc? If yes, should I change the SO_VERSION including the package version numbers already in this PR?

Reply:
Exactly, I mean this major shared object (SO) number for the core library which is both in the package names (like libelektra**4**-all) and in the shared objects (as defined by SO_VERSION).

It is up to you in which PR you want to do it.

Reply:
I suppose the installation of of a libelektra5-* package should break/replace libelektra4-* packages? Or should it be possible to install both? (Unfortunately I'm not that familiar with SO versions.)

Reply:
I am not an expert in Debian packaging but I think you are right, e.g.:

```
apt-cache show libpurelibc1                                                                                                                                                                             Package: libpurelibc1
Source: purelibc
Version: 0.4.1-2
Installed-Size: 74
Maintainer: Debian VSquare Team <virtualsquare@cs.unibo.it>
Architecture: amd64
Replaces: libpurelibc0
Depends: libc6 (>= 2.14)
Conflicts: libpurelibc0
```

So a "Replaces:" and "Conflicts:" seems to be the right thing.

> Or should it be possible to install both?

This sounds like a lot of effort, I do not think this is needed.